governance/polls/templates/MIP10c14-SP$sp_number: Appoint $appointed_entity as Oracle Light Feed.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Appoint $appointed_entity as Oracle Light Feed (MIP10c14-SP$sp_number) - $date_MONTH_DD,_YYYY
```

Comment:
Nice work setting this up! Based on my suggestions for this week's poll I imagine anything you change there you will probably want to change here as well, unless we want to play around with the wording in this doc

Reply:
Done!